webcompat/api/uploads.py

```diff
@@ -83,9 +83,15 @@ def save(self):
             raise TypeError('Image file format not allowed')
 
         today = date.today()
-        self.file_path = os.path.join(str(today.year), str(today.month),
-                                      str(uuid4()) + '.' + self.file_ext)
+        year = str(today.year)
+        month = str(today.month)
+        image_id = str(uuid4())
+        self.file_path = os.path.join(year, month,
+                                      image_id + '.' + self.file_ext)
+        self.thumb_path = os.path.join(year, month,
+                                       image_id + 'thumb.' + self.file_ext)
```

Comment:
This does feel like we're repeating ourselves here. Not sure a nicer way to do it though... thinking out loud here:

```
def get_image_path(img_type=''):
    today = date.today()
    year = str(today.year)
    month = str(today.month)
    return os.path.join(year, month, image_id + '{img_type}'.format(img_type=img_type)  + 
      self.file_ext)
```

get_img_path(img_type=thumb)
get_img_path()

I guess then get rid of the file_path and thumb_path properties. Maybe. Not sure. 🎱